components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/model/TokenIssuanceDO.java

```diff
@@ -0,0 +1,184 @@
+/*
+ *  Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.identity.oauth2.model;
+
+import org.wso2.carbon.identity.openidconnect.OIDCConstants;
+
+/**
+ * Data object to contain information related to token issuance.
+ */
+public class TokenIssuanceDO {
```

Comment:
We can use event properties rather than using this object. Check with other use cases.

Reply:
Object removed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2815/commits/04edb09757c15b2999491db476f1b7be8ad1fee0